<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dettagli_prod_dialog</class>
 <widget class="QDialog" name="dettagli_prod_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>366</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>578</width>
    <height>366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dettagli prodotto</string>
  </property>
  <widget class="QLabel" name="lbl_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dettagli del prodotto</string>
   </property>
  </widget>
  <widget class="QLabel" name="prod_image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;prod_image&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prod_title">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;prod_title&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prod_price">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;prod_price&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prod_details">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>149</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID: &lt;prod_id&gt; - &lt;prod_qta&gt; pezzi rimasti in magazzino</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prod_description">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&lt;prod_description&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Chiudi</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
